Addons/AvatarOnlyMemberList.css

```diff
@@ -31,7 +31,12 @@ div:not(:hover) > .members_cbd271 > div {
     transition: 0.5s ease;
 }
 
-.members_cbd271:hover {
+/* overwrite black stuff for member list */
+.theme-light .members_cbd271:hover {
+    background-color: rgba(255, 255, 255, 0.87);
```

Comment:
Please add `margin-left`
![image](https://github.com/user-attachments/assets/effd95ff-562c-403f-8b09-39ef08233cbb)

Reply:
This is the only bug that I found after testing for a while. Just add the `margin-left` with the default value and it will be good to go.